frontend/src/app/databases/databases-list.tsx

```diff
@@ -24,6 +25,16 @@ export default function DatabasesList() {
     mutate: mutateDatabases,
   } = useDatabases();
 
+  const [filteredDatabases, setFilteredDatabases] = useState<CleanDatabase[]>(
+    [],
+  );
+
+  useEffect(() => {
+    setFilteredDatabases(
+      databases?.filter((database) => !database.deleted_at) || [],
+    );
+  }, [databases]);
```

Comment:
Our use databases hook should only return non-deleted databases, so I don't think we need this useEffect